apps/06.1Checkers.js

```diff
@@ -61,6 +128,51 @@ function Game() {
   this.start = function() {
     this.board.createGrid();
     // Your code here
+    this.board.createCheckers();
+  };
+  //start might look like 50
+  //end might look like 41
+  this.moveChecker = function(start, end){
+    //Inside the method, use your board helper method selectChecker to select the checker at your starting rowcolumncoordinates and set it to a local variable checker. Then set that spot on the grid to null and set the spot at the end rowcolumn coordinate to the checker.
+    var startRow = start.split("")[0];
+    var startColumn = start.split("")[1];
+    var endRow = end.split("")[0];
+    var endColumn = end.split("")[1];
+    var checker = this.board.selectChecker(startRow, startColumn);
+    this.board.grid[startRow][startColumn] = null;
+    this.board.grid[endRow][endColumn] = checker;
+    var distance = Math.abs(startRow - endRow);
+    // var killX = Math.abs(startRow - endRow);
+    // var killY = Math.abs(startColumn - endColumn);
+    // var killPosition = (killX+killY);
+    if(distance === 2){
+      var killRow =(parseInt(startRow) + parseInt(endRow)) / 2; 
+      var killColumn = (parseInt(startColumn) + parseInt(endColumn)) / 2;
+      var killPosition = [killRow, killColumn];
+      this.board.killChecker(killPosition);
+    }
```

Comment:
All tests passed!

![dab](https://media.giphy.com/media/l3zoKeX8bMG5sMP4s/giphy.gif)

***Make sure to remove the code you commented out before you host this on your portfolio.